deploy/configure.sh

```diff
@@ -39,14 +45,14 @@ fi
 if [ -z "$NAMESPACE" ]; then
   echoerr "WARN: Recommended setting 'NAMESPACE' variable (default: pixel)"
 fi
-if [ -z "$OCIR_SECRET" ]; then
```

Comment:
Why need OCIR secret? Cannot use a regular docker config secret to auto auth ocir.io?

Reply:
right.. this is a regular `imagePullSecret` ... I've just changed naming to be more generic